app/controllers/errors_controller.rb

```diff
@@ -1,4 +1,6 @@
 class ErrorsController < ApplicationController
+  alias_method :current_user, :current_member
```

Comment:
stiu asta, dar daca las in application_controller nu porneste serverul.. zice undefined method, dar asa merge